docs/spatial-raster.rst

```diff
@@ -7,13 +7,13 @@ and ``geopandas`` to work with raster and vector data in Python.
 Stack Raster Files
 ~~~~~~~~~~~~~~~~~~
 
-The ``stack_raster_tifs`` function takes a list of raster paths and turns that list
-into an
+The ``stack_raster_tifs`` function takes a list of raster paths and turns the list
+into the following:
 
-1. a stacked geotiff on your hard drive and
-2. (optionally) an output raster stack in numpy format with associated metadata.
+1. a stacked geotiff on your local hard drive and
```

Comment:
i'd say just on your hard drive because this isn't necessarily local! it could be run in a cloud environment.